### PR TITLE
Adding docker_container_fs_changes table

### PR DESCRIPTION
**Pull request by scoders-tob**
This table lists changes to a container's filesystem since it's creation.

Example query:

```
osquery> select * from docker_container_fs_changes where id = '6d4a0ff6d954';
+--------------+----------+-------------+
| id           | path     | change_type |
+--------------+----------+-------------+
| 6d4a0ff6d954 | /bin     | C           |
| 6d4a0ff6d954 | /bin/xxx | A           |
| 6d4a0ff6d954 | /bin/tar | D           |
| 6d4a0ff6d954 | /xxx     | A           |
+--------------+----------+-------------+
```
change_type legend: C:Modified, A:Added, D:Deleted

### CODE REVIEW

File: tools/tests/test_docker_container_fs_changes_table.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        except docker_exceptions as e:
+            print(f"Failed in creating test container: {e}") 
+
+    def tearDown(self):
+        if self.container:
+            self.container.stop()
+        if self.client:
+            self.client.close()
+    
+    def testFsChangesTable(self):
```
Comment (theopolis) [inline tools/tests/test_docker_container_fs_changes_table.py]: If someone runs the test and they do not have `docker` installed then the test will print the error message from `setUp` but then expect that `self.container` is usable. This mean we require `docker` for tests to run.

Comment (theopolis) [reply tools/tests/test_docker_container_fs_changes_table.py]: I am not sure if this is a reasonable expectation for people developing osquery, should we expect developers to have `docker` installed?

Comment (scoders-tob) [reply tools/tests/test_docker_container_fs_changes_table.py]: Makes sense, will fix it, thanks!

Comment (scoders-tob) [reply tools/tests/test_docker_container_fs_changes_table.py]: Done

**Comment by scoders-tob** [comment]
Done!

**Comment by scoders-tob** [comment]
Hi @theopolis, request to merge this one please if you're okay with it.

**Comment by theopolis** [comment]
@Smjert or @alessandrogario can you merge this when you are ready. I don't want to be the one who merges due to the Azure image update coordination.

**Comment by Smjert** [comment]
> @Smjert or @alessandrogario can you merge this when you are ready. I don't want to be the one who merges due to the Azure image update coordination.

You can go ahead with this, the Docker image name is correct, Sahil already did that, thanks!

**Pull request merged**